crates/sui-core/src/authority_active/checkpoint_executor/mod.rs

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use core::panic;
+use std::{sync::Arc, time::Duration};
+
+use broadcast::Receiver;
```

Comment:
This is fairly confusing as this is referencing the tokio broadcast module. I'd remove this import and just directly qualify Recevier below with `broadcast`